__tests__/browser.ts

```diff
@@ -84,10 +84,10 @@ describe('browser', () => {
         try {
           await client.bzz.download('abcdef123456')
         } catch (err) {
-          return err.message
+          return err.status
```

Comment:
Why are you changing the returned error here please?

Reply:
Well, it is generally better to assert on stable elements, which error messages are not as they might change. That is for example why Nodejs has [`code`](https://nodejs.org/api/errors.html#errors_error_code) property. In HTTP `status` is a great way for that as well. I would not dare to assert on `400` as it can mean pretty much anything (in Swarm), but I think `404` in this context is fairly usable. 

I can revert the change if you would like.

Reply:
Well, now I realize that was only a partial answer. Now with the exposed error messages, you get actually very dynamic messages that contain the hash (something like "hash <hash> not found") so instead of doing some regex match, I switched to this assert.

Reply:
OK makes sense, thanks!